doc-surrealdb_versioned_docs/version-latest/querying/graphql/surrealist.mdx

```diff
@@ -0,0 +1,137 @@
+---
+sidebar_position: 1
+sidebar_label: GraphQL via Surrealist 
+title: GraphQL via Surrealist | GraphQL
+description: In this section, you will explore interating with your SurrealDB instance using GraphQL queries using Surrealist, the official query editor for SurrealDB. Surrealist is a powerful tool that allows you to write, execute, and visualize GraphQL queries in real-time.
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# GraphQL via Surrealist 
+
+In this section, you will explore writing GraphQL queries in [Surrealist](https://surrealist.app/query). The GraphQL query view in Surrealist provides a rich set of features, including syntax highlighting, query validation, and query execution. You can see the results of your queries in JSON structure returned by GraphQL.
+
+## Getting Started
+
+Before you can start making queries, you need to start SurrealDB with the GraphQL module enabled. You can do this by setting the `SURREAL_EXPERIMENTAL_GRAPHQL` [environment variable](/docs/surrealdb/installation/running/env_vars) to `true` and starting a new instance of SurrealDB with the [`surreal start` command](/docs/surrealdb/cli/start).
+
+```bash
+SURREAL_EXPERIMENTAL_GRAPHQL=true surreal start --log debug --user root --password root
+```
+
+:::note
+__Note:__ By running SurrealDB with the GraphQL module enabled, you are opting into an experimental feature. In SurrealDB GraphQL is still considered experimental and may not be as stable as querying via SurrealQL.  
+
+GraphQL is a pre-release feature with known security flaws. This is not recommended for production use. We welcome your feedback and contributions to help improve the feature and make it more robust.
+:::
+
+After starting the SurrealDB instance, you can navigate to the Surrealist to start a new connection. 
+
+### Start a new connection
+
+In the top left corner of the Surrealist, start a new connection. Ensure that the connection information is the same as the one you used to start the SurrealDB instance. In the example above we have set the user to `root` and the password to `root`. 
+
+:::note
+__Note:__ In a sandbox connection, querying via GraphQL is not supported. 
+:::
+
+Learn more about starting a connection in the [Surrealist documentation](/docs/surrealist/getting-started). 
+
+### Setting a namespace and database
+
+Before you can start writing queries, you need to set the [namespace](/docs/surrealdb/introduction/concepts/namespace) and [database](/docs/surrealdb/introduction/concepts/database) you want to use. For example you can set the namespace to `test` and the database to `test`. This will set the namespace and database for the current connection.
+
+Additionally, you can start [a serving in Surrealist](/docs/surrealist/concepts/local-database-serving) which also enables graphql automatically, this starts a server on `http://localhost:8000` by default for a root user with username and password `root`. 
```

Comment:
graphql -> GraphQL, I think

Also would recommend changing "this" to "which" or starting a new sentence with the existing "this".